packages/web3-eth-ens/src/lib/contentHash.js

```diff
@@ -0,0 +1,85 @@
+/*
+  Adapted from ensdomains/ui
+  Permalink: https://github.com/ensdomains/ui/blob/3e62e440b53466eeec9dd1c63d73924eefbd88c1/src/utils/contents.js#L1-L85
+*/
```

Comment:
Have opened an issue at [@ensdomains/ui 62][1] asking for clarification about how to license & attribute this file's contents appropriately. The repo currently has no license. 

[1]: https://github.com/ensdomains/ui/issues/62